crate_universe/src/metadata/cargo_tree_rustc_wrapper.bat

```diff
@@ -7,15 +7,15 @@ SETLOCAL ENABLEDELAYEDEXPANSION
 
 @REM When cargo is detecting the host configuration, the host target needs to be
 @REM injected into the command.
-echo %*| FINDSTR /C:"rustc.exe - --crate-name ___ " | FINDSTR /V /C:"--target" >NUL
+echo %*| FINDSTR /R /C:".*rustc[\.exe\"\"]* - --crate-name ___ " | FINDSTR /V /C:"--target" >NUL
```

Comment:
bzlmod introduced the need for quotes in the command line which broke the rustc wrapper on windows.